app/src/main/java/org/mozilla/fenix/home/sessioncontrol/RecentTabsListFeature.kt

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.sessioncontrol
```

Comment:
nit: Probably meant for this (and it's test) to be in `org.mozilla.fenix.home.recenttab`.

Reply:
Will do this as a fast follow to not lose the green CI